src/govuk/common/normalise-dataset.mjs

```diff
@@ -1,5 +1,5 @@
 import '../vendor/polyfills/Element/prototype/dataset.mjs'
-import '../vendor/polyfills/String/prototype/trim.mjs'
+import '../vendor/polyfills/String/prototype/trim.mjs' /* eslint-disable es-x/no-string-prototype-trim */
```

Comment:
I'm surprised this works as my understanding is this [only disables the rule for this line](https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-comments-1:~:text=To%20disable%20a%20specific%20rule%20on%20a%20specific%20line%3A) – and I'd expect the linting rule to be triggered where we're calling `String.prototype.trim` rather than at the point we're importing the polyfill? 🤔 

(Same for everywhere else we're doing this)

Reply:
Apparently I can't read – that example clearly uses `eslint-disable-line` 🤦🏻 

I think you can probably ignore this, but wonder if others might get tripped up on what this is doing.